README.md

```diff
@@ -1,13 +1,14 @@
-# Welcome to Open Source Maintainers on GitHub!
-The GitHub maintainer community is a place for some open source maintainers on GitHub to collaborate with each other, and with GitHub.
+# Welcome to Open Source Maintainers at GitHub!
```

Comment:
that change makes it sound like it’s for maintainers who work at GitHub, compared to maintainers that use GitHub

Reply:
Good idea! 
```suggestion
# Welcome to the Open Source Maintainers organization
```